Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -230,6 +230,13 @@ CookieStore::~CookieStore()
     m_mainThreadBridge->detach();
 }
 
+static URL creationURL(ScriptExecutionContext& context)
```

Comment:
Can probably use `const ScriptExecutionContext&`

Reply:
Changed to using cookieURL() everywhere, so we don't need this function anymore.